Dockerfile

```diff
@@ -139,7 +139,7 @@ ARG FIN_TOKEN=
 RUN git clone https://$FIN_TOKEN:x-oauth-basic@github.com/ROCmSoftwarePlatform/Fin.git $FIN_DIR
 WORKDIR $FIN_DIR
 # Can be a branch or a SHA
-ARG FIN_BRANCH=39024934b61177d3f719f206bc2445b40edf4db7
+ARG FIN_BRANCH=29ddb2d4969d1a81f0d1c48fc6384f1a818d14d7
```

Comment:
I dont see this commit in the develop branch? where is this hash coming from? Can u point it to the latest in develop?